cmd/operator/main.go

```diff
@@ -121,7 +121,9 @@ func init() {
 	// Prometheus Operator image, tagged with the same semver version. Default to
 	// the Prometheus Operator version if no Prometheus config reloader image is
 	// specified.
-	flagset.StringVar(&cfg.PrometheusConfigReloader, "prometheus-config-reloader", fmt.Sprintf("quay.io/coreos/prometheus-config-reloader:v%v", version.Version), "Prometheus config reloader image")
+	flagset.StringVar(&cfg.PrometheusConfigReloaderImage, "prometheus-config-reloader", fmt.Sprintf("quay.io/coreos/prometheus-config-reloader:v%v", version.Version), "Prometheus config reloader image")
+	flagset.StringVar(&cfg.PrometheusConfigReloaderCPU, "prometheus-config-reloader-cpu", "50m", "Prometheus config reloader CPU")
```

Comment:
given that these sidecars work almost identically, how about we just use the same flag for this?

Reply:
Same flag being just using the `config-reloader-cpu` and `config-reloader-memory` for both the `config-reloader` and `prometheus-config-reloader` containers?

Reply:
yes, that's what I was thinking

Reply:
I'd be good with that. I'll go ahead and change that.